packages/element/src/create-interpolate-element.ts

```diff
@@ -53,27 +70,27 @@ const tokenizer = /<(\/)?(\w+)\s*(\/)?>/g;
  * parsed.
  *
  * @private
- * @param {Element} element            A parent element which may still have
- *                                     nested children not yet parsed.
- * @param {number}  tokenStart         Offset at which parent element first
- *                                     appears.
- * @param {number}  tokenLength        Length of string marking start of parent
- *                                     element.
- * @param {number}  [prevOffset]       Running offset at which parsing should
- *                                     continue.
- * @param {number}  [leadingTextStart] Offset at which last closing element
- *                                     finished, used for finding text between
- *                                     elements.
+ * @param {ReactElement}  element            A parent element which may still have
+ *                                           nested children not yet parsed.
+ * @param {number}        tokenStart         Offset at which parent element first
+ *                                           appears.
+ * @param {number}        tokenLength        Length of string marking start of parent
+ *                                           element.
+ * @param {number}        [prevOffset]       Running offset at which parsing should
+ *                                           continue.
+ * @param {number | null} [leadingTextStart] Offset at which last closing element
+ *                                           finished, used for finding text between
+ *                                           elements.
  *
  * @return {Frame} The stack frame tracking parse progress.
```

Comment:
Let us remove the types from JSDoc

```suggestion
 * @param element          A parent element which may still have
 *                         nested children not yet parsed.
 * @param tokenStart       Offset at which parent element first
 *                         appears.
 * @param tokenLength      Length of string marking start of parent
 *                         element.
 * @param prevOffset       Running offset at which parsing should
 *                         continue.
 * @param leadingTextStart Offset at which last closing element
 *                         finished, used for finding text between
 *                         elements.
 *
 * @return  The stack frame tracking parse progress.
```